modules/standard/Time.chpl

```diff
@@ -403,28 +403,28 @@ enum day       { sunday=0, monday, tuesday, wednesday, thursday, friday, saturda
   }
 
   /* Return the number of days since 1-1-0001 this `date` represents */
-  proc date.toOrdinal() {
+  proc date.toOrdinal() : int {
     return ymdToOrd(year, month, day);
   }
 
   /* Return the day of the week as a `dayOfWeek`.
      `Monday` == 0, `Sunday` == 6
    */
-  proc date.weekday() {
+  proc date.weekday() : dayOfWeek {
     // January 1 0001 is a Monday
     return try! ((toOrdinal() + 6) % 7): dayOfWeek;
   }
 
   /* Return the day of the week as an `isoDayOfWeek`.
      `Monday` == 1, `Sunday` == 7 */
-  proc date.isoWeekday() {
+  proc date.isoWeekday() :isoDayOfWeek {
```

Comment:
nitpick, but there should be a space here to match the style of the rest of the return types